test/langtools/tools/javac/reflect/TestIRFromAnnotation.java

```diff
@@ -65,11 +65,11 @@ public class TestIRFromAnnotation {
             "SuperTest.java",                   // @@@ Might be issue referencing auxiliary interface
                                                 // in method superInterfaceMethodInvocation
             "LocalClassTest.java",              // name of local classes is not stable at annotation processing time
-            "TestLocalCapture.java",            // plain testng test
-            "TestCaptureQuoted.java",           // plain testng test
-            "TestCaptureQuotable.java",         // plain testng test
-            "QuotedSameInstanceTest.java",      // plain testng test
-            "CodeModelSameInstanceTest.java"    // plain testng test
+            "TestLocalCapture.java",            // plain junit test
```

Comment:
When looking again at this, it would perhaps be beneficial to put all the IR tests in a separate folder. It seems a bit sad that whenever we add a "plain" test we have to come back and list it here.

The exclusion for `SuperTest`/`LocalClassTest` of course make more sense, so it's ok for them to stay.

Reply:
Actually, "plain" test should be in the runtime tests directory.